spring-kafka/src/main/java/org/springframework/kafka/support/micrometer/KafkaRecordReceiverContext.java

```diff
@@ -28,16 +28,43 @@
  * {@link ReceiverContext} for {@link ConsumerRecord}s.
  *
  * @author Gary Russell
+ * @author Christian Mergenthaler
+ * @author Wang Zhiyang
+ *
  * @since 3.0
  *
  */
 public class KafkaRecordReceiverContext extends ReceiverContext<ConsumerRecord<?, ?>> {
 
 	private final String listenerId;
 
+	private final String clientId;
+
+	private final String groupId;
+
 	private final ConsumerRecord<?, ?> record;
 
+	/**
+	 * Construct a kafka record receiver context.
+	 * @param record 		the consumer record.
+	 * @param listenerId	the container listener id.
+	 * @param clusterId		the kafka cluster id.
+	 */
 	public KafkaRecordReceiverContext(ConsumerRecord<?, ?> record, String listenerId, Supplier<String> clusterId) {
+		this(record, listenerId, null, null, clusterId);
+	}
+
+	/**
+	 * Construct a kafka record receiver context.
+	 * @param record 		the consumer record.
+	 * @param listenerId	the container listener id.
+	 * @param clientId		the kafka client id.
+	 * @param groupId		the consumer group id.
+	 * @param clusterId		the kafka cluster id.
+	 * @since 3.2
+	 */
+	public KafkaRecordReceiverContext(ConsumerRecord<?, ?> record, String listenerId, String clientId, String groupId,
```

Comment:
New ctor, so `@since 3.2`

Reply:
Already have.

Reply:
Thanks.

If you agree with me, there is no need to say that via `Done/Fixed/OK` comments.
I'll see the change I've asked before in your next commit 😄

Reply:
Thanks! You are right!